corehq/apps/userreports/reports/view.py

```diff
@@ -522,6 +522,10 @@ def sub_slug(self):
         """
         return self.report_config_id
 
+    @property
+    def html_id_suffix(self):
+        return f"-{self.report_config_id}"
```

Comment:
how does this look then for usual reports?
the id starts with a "-"? Is that a bit unusual? how about having a default here like "report" instead of empty string?

Reply:
How does this get passed along to the context? I see the view's `self` is included in the context [here](https://github.com/dimagi/commcare-hq/blob/66d0a8240d3f962b08e6693f8bcae08eac81ba78/corehq/apps/userreports/reports/view.py#L349), does that mean both properties and attributes get included?

Reply:
@mkangia I believe standard reports won't have a suffix, because of the change to `corehq/apps/reports/generic.py` that passes a blank string.

As an alternative to Manish's suggestion, the js could have a util function like `getReportElementId(prefix)` that would check initial page data and return the full id, or just return the prefix if no suffic is found in initial page data.

Reply:
oh, noticed it now that it was suffix and not prefix. So this is appended to the end. All okay :+1: